packages/beacon-node/package.json

```
@@ -76,10 +76,10 @@
     "lint": "eslint --color --ext .ts src/ test/",
     "lint:fix": "yarn run lint --fix",
     "test": "yarn test:unit && yarn test:e2e",
-    "test:unit:minimal": "vitest --run --segfaultRetry 3 --dir test/unit/",
-    "test:unit:mainnet": "LODESTAR_PRESET=mainnet vitest --run --dir test/unit-mainnet",
-    "test:unit": "wrapper() { yarn test:unit:minimal $@ && yarn test:unit:mainnet $@; }; wrapper",
-    "test:e2e": "LODESTAR_PRESET=minimal vitest --run --segfaultRetry 3 --config vitest.e2e.config.ts --dir test/e2e",
+    "test:unit:minimal": "LODESTAR_PRESET=minimal vitest --run --segfaultRetry 3 --dir test/unit/",
+    "test:unit:mainnet": "vitest --run --segfaultRetry 3 --dir test/unit-mainnet",
```

this command is mainnet preset specific, I don't think it is good that this can be executed with a different preset if the value in .env.test is changed

```suggestion
    "test:unit:mainnet": "LODESTAR_PRESET=mainnet vitest --run --segfaultRetry 3 --dir test/unit-mainnet",
```

---

Don't need mainnet preset as it will be passed from the env file.

---

> Don't need mainnet preset as it will be passed from the env file.

But I don't get the purpose of the value set in .env file to be overriden by a user? I don't get the purpose of the .env file if it not that the user can easily override the value within